src/openms/source/TRANSFORMATIONS/RAW2PEAK/PeakShape.cpp

```diff
@@ -53,31 +53,32 @@ namespace OpenMS
     right_endpoint_ = exp_.end();
   }
 
-  PeakShape::PeakShape(double height, double mz_position, double left_width, double right_width, double area, PeakIterator left, PeakIterator right, Type type) :
-    height(height),
-    mz_position(mz_position),
-    left_width(left_width),
-    right_width(right_width),
-    area(area),
+  PeakShape::PeakShape(double height_, double mz_position_, double left_width_, 
+      double right_width_, double area_, PeakIterator left_, PeakIterator right_, Type type_) :
+    height(height_),
```

Comment:
I used XY_arg

strangely, the members are public and therefore dont have a  trailing underscore